doc/news/changes/minor/20231115SebastianKinnewig

```diff
@@ -0,0 +1,5 @@
+New: Introduce a new function, Utilities::Trilinos::make_rcp(),
+to create Teuchos::RCP objects, avoiding the usage of
+'operator new'.
+<br>
+(Sebastian Kinnewig, 2023/11/15)
```

Comment:
You'll need an endline at the end of this line.

Reply:
Done!